src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -92,12 +99,16 @@ public CursorsController(final CursorsService cursorsService,
 
         try {
             validateSubscriptionReadScopes(client, subscriptionId);
-            final Map<SubscriptionCursor, Boolean> result = cursorsService.commitCursors(streamId, subscriptionId,
+
+            LOG.debug("[COMMIT_CURSORS] scopes validation finished");
+
+            final List<CursorCommitResult> items = cursorsService.commitCursors(streamId, subscriptionId,
                     cursors.getItems());
-            final List<CursorCommitResult> items = result.entrySet().stream()
-                    .map(entry -> new CursorCommitResult(entry.getKey(), entry.getValue()))
-                    .collect(Collectors.toList());
-            final boolean allCommited = result.values().stream().reduce(Boolean::logicalAnd).orElseGet(() -> true);
+
+            LOG.debug("[COMMIT_CURSORS] commit finished");
```

Comment:
Each log message already has a timestamp. 
Also, all these "[COMMIT_CURSORS] ..." logs are temporary and will be removed after the performance problem is resolved.